src/howTos/dev/services.md

```diff
@@ -15,6 +15,14 @@ In contrast to konnectors, services have the same permissions as the web applica
 
 You can find an example of an existing service in the [cozy-banks app](https://github.com/cozy/cozy-banks/blob/master/src/targets/services/onOperationOrBillCreate.js).
 
+## CozyClient instanciation 
+
+By using `fromEnv`, you will be able to use the service in dev mode (via cozy-konnector-dev see [Execution](#execution)) or in production. 
+```js
```

Comment:
Can you add a blank line before the code block ? Depending on the markdown parser, I think this can cause problems (problem being that the rendering is OK in GitHub and not on docs.cozy.io).